docusaurus.config.js

```diff
@@ -41,7 +41,10 @@ const config = {
           customCss: require.resolve("./src/css/custom.scss"),
         },
         gtag: {
-          trackingID: process.env.GOOGLE_TAG_MANAGER_TRACKING_ID || "gtm-dev",
+          trackingID: process.env.GOOGLE_TAG_TRACKING_ID || "gtag-dev",
+        },
+        googleTagManager: {
+          containerId: process.env.GOOGLE_TAG_MANAGER_CONTAINER_ID || "gtm-dev",
         },
```

Comment:
From what I see, it won't be enabled anyway in the `npm run start` mode 🤔 
![Screenshot 2023-02-20 at 14 02 40](https://user-images.githubusercontent.com/17568639/220116205-6c5f1bcb-5e6b-4546-980c-2aa0b253b324.png)

but I'm not against defaults, maybe it would be even beneficial to have `GTM-tag-dev` and `trackingID: 'G-tracking-dev'` just to have a proper prefixes in place in case someone will need to put a proper values there.

Reply:
It won't be, but we will have defaults when building and serving website locally or on Cloudflare Pages preview. I will add the prefixes 👍